### ISSUE TITLE
Nuitka (v0.6.9.2) error with --windows-icon=ICON_PATH

### ISSUE THREAD

**Issue by Alpengreis** [Nuitka/Nuitka#855]
Hello

I receive always an error with --windows-icon=ICON_PATH ...

![nuitka_icon-problem](https://user-images.githubusercontent.com/11355601/95487581-cc294a80-0994-11eb-93a3-bca7522b71c2.png)

As text (not as code because newline chars):

"d:\_D-Alpen\Multimedia\PC\Python\Projects\.venv\py3.8.6\Scripts\python.exe" -m nuitka --pl
ugin-enable=tk-inter --standalone --windows-disable-console --windows-dependency-tool=pefile --experimental=use_pefile_r
ecurse --experimental=use_pefile_fullrecurse --windows-icon="src\program\images\test.ico" "src\program\frames_pack_gui.p
yw"
Nuitka-Plugins:INFO: tk-inter: Injecting pre-module load code for module 'tkinter':
Nuitka-Plugins:INFO: tk-inter:     Need to make sure we set environment variables for TCL.
gcc: error: resources.rc: No such file or directory
gcc: warning: '-x c' after last input file has no effect
gcc: fatal error: no input files
compilation terminated.
windres: preprocessing failed.
scons: *** [resources.o] Error 1

* Nuitka version, full Python version and Platform (Windows, OSX, Linux ...)

  ~~python -m nuitka --version~~
  ~~0.6.9.2~~
  ~~Python: 3.9.0 (tags/v3.9.0:9cf6752, Oct  5 2020, 15:34:40) [MSC v.1927 64 bit (AMD64)]~~
  ~~Executable: C:\Program Files\Python39\python.exe~~
  ~~OS: Windows~~
  ~~Arch: x86_64~~

  python -m nuitka --version
  0.6.9.2
  Python: 3.8.6 (tags/v3.8.6:db45529, Sep 23 2020, 15:52:53) [MSC v.1927 64 bit (AMD64)]
  Executable: C:\Program Files\Python38\python.exe
  OS: Windows
  Arch: x86_64

  Microsoft Windows 10 Pro x64 [19041.508] [2004] [de-CH]

* How did you install Nuitka and Python (pip, anaconda, deb, rpm, from source,
  what is a virtualenv ...), this is very important usually.
  ~~pip install -U nuitka~~
  pip install nuitka
  --> within restricted account, so it was an user installation

Edit: added text data and made some correction.

**Comment by kayhayen** [comment]
Btw, screenshots are not welcome, because they suck at copy&past, search, and reading for blind people. I should make that clear in the issue template.

**Comment by kayhayen** [comment]
Your report is inconsistent, it says you were using Python39 with 0.6.9.2, which I know for a fact doesn't work at all until 0.6.9.3 later today. 

And the environment that leads to using gcc is also not given, but there is a gcc error message. If I need to bet, I would say you didn't read the user manual and just used any gcc that you found. In part, MinGW64 project is to blame, because they keep changing their downloads, and only one of them is really good, so you may also be victim of that. I updated user manual on factory branch a couple of days ago, to be more clear about what to pick. 

Using a specific gcc on Windows is hit or miss. We might have to make Nuitka download things for itself, or something like that, just like I recently added it for `ccache.exe`.

**Comment by Alpengreis** [comment]
- About screenshots: ok, I understand and I see the problems with, thank you.

- The exact same problem is also with Python38 and 0.6.9.2.

- About gcc:: I read the manual, honestly! Maybe you can't imagine how many new things are new for a beginner. I would say, It's not soooo easy.

About the gcc environment: it's my bad: here the data:

gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=c:/program\ files/mingw64/bin/../libexec/gcc/x86_64-w64-mingw32/9.2.0/lto-wrapper.exe
Target: x86_64-w64-mingw32
Configured with: ../src/configure --enable-languages=c,c++ --build=x86_64-w64-mingw32 --host=x86_64-w64-mingw32 --target=x86_64-w64-mingw32 --disable-multilib --prefix=/c/temp/gcc/dest --with-sysroot=/c/temp/gcc/dest --disable-libstdcxx-pch --disable-libstdcxx-verbose --disable-nls --disable-shared --disable-win32-registry --with-tune=haswell --enable-threads=posix --enable-libgomp
Thread model: posix
gcc version 9.2.0 (GCC)

I took this distro:

https://nuwen.net/mingw.html

maybe that was not correct? Could you help me, which download I should use, please?

Thanks!

**Comment by Alpengreis** [comment]
Oh, this problem is solved!

The reason was that I used the alternative command line interpreter TCC (Take Command) instead original command prompt. If I use the original command prompt, it runs correctly:

As text (not as code because newline chars):

"d:\_D-Alpen\Multimedia\PC\Python\Projects\.venv\py3.8.6\Scripts\python.exe" -m nuitka --plugin-enable=tk-inter --standalone --windows-disable-console --windows-dependency-tool=pefile --experimental=use_pefile_recurse --experimental=use_pefile_fullrecurse --windows-icon="src\program\images\test.ico" "src\program\frames_pack_gui.pyw"
Nuitka-Plugins:INFO: tk-inter: Injecting pre-module load code for module 'tkinter':
Nuitka-Plugins:INFO: tk-inter:     Need to make sure we set environment variables for TCL.
Nuitka-Scons:INFO: Compiled 13 C files using ccache.
Nuitka-Scons:INFO: Cached C files (using ccache) with result 'cache hit (direct)': 13
Nuitka-Plugins:INFO: tk-inter: Copied Tk libraries from C:\Program Files\Python38\tcl\tk8.6.
Nuitka-Plugins:INFO: tk-inter: Copied TCL libraries from C:\Program Files\Python38\tcl\tcl8.6.

I never thought that this could be the reason!

So, I have to apologize. The only good thing is that it could help other users with same or similar problem(s).
